test/test-signal.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+// Testcase to produce NodeReport on signal interrupting a js busy-loop, 
+// showing it is interruptible.
+if (process.argv[2] === 'child') {
+  require('../').setEvents("signal");
+
+  // Exit on loss of parent process
+  process.on('disconnect', () => process.exit(2));
+
+  function busyLoop() {
+    var list = [];
+    for (var i=0; i<10000000000; i++) {
```

Comment:
Spaces around `=` and `<` and maybe write the large number as 1e10 for better legibility.